internal/compiler/tests/syntax/basic/path_commands.slint

```diff
@@ -24,13 +24,22 @@ export TestCase := Rectangle {
         commands: "M 0 0 M -100 0 A 100 100 0 1 0 100 0 A 100 100 0 1 0 100 0 Z";
     }
 
-    Path {
+    p := Path {
         width: 640px;
         height: 480px;
         commands: "M 0 0 M -100 0 A 100 100 0 1 0 0 A 100 100 0 1 0 100 0 Z";
         //        ^error{Error parsing SVG commands}
     }
 
+    TouchArea {
+        clicked => {
+            p.commands = "M 0 0 M -100 0 A 100 100 0 1 0 100 0 A 100 100 0 1 0 100 0 Z";
```

Comment:
I'm confused about this part. WesAudio (and others) set the commands property dynamically at run-time from C++. If that works, then... this should also be okay, shouldn't it?

It's the reading part that I don't see making any sense.

Reply:
They probably do something like
```
commands: SomeGlobal.the-command;
```

As a binding.

The assigning currently doesn't work and either panic the interpreter or the generated code doesn't compile. (issue #5564)

We would need to rewrite the expression for that to work.

Reply:
Or `p.commands = "..."` could just assign `PathData::Commands("...")` ?